api_specs/v2.2/component_analyses_v2.2.yaml

```diff
@@ -162,6 +162,7 @@ components:
   examples:
     higher-rate-3scale:
       value:
+        - url: "https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2/"
```

Comment:
are you not adding the source here too?

Reply:
It is an example, actual value will be returned from code only.

Reply:
@yzainee I am not sure what would be the use of keeping the source field.